packages/tools/test/getVolumeId_test.js

```diff
@@ -0,0 +1,13 @@
+import { getVolumeId } from '../../tools/src/utilities/getVolumeId';
+
+describe('getVolumeId', () => {
+  it('should extract volumeId from target', () => {
+    const targetId = 'volumeId:aaaa';
+    expect(getVolumeId(targetId)).toEqual('aaaa');
+  });
+
+  it('should extract volumeId from a complex nifti url', () => {
+    const targetId = 'nifti:https://nifti.com?xoxo';
```

Comment:
Are you sure this one isn't  'volumeId:nifti:https://nifti.com?xoxo'   Otherwise this would not have worked previously since
'nifti:https://nifti.com?xoxo'.split(/volumeId:/)[1] 
returns undefined

Reply:
At which moment "volumeId:" is added?